### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by harelm4**

### CODE REVIEW

File: src/model/ensemble.py [hunk 1]
```diff
@@ -40,35 +40,37 @@ def predict(self, text: str, min_p=0.1) -> EnsembleResult:
...
+        :param pred_index: index of prediction
+        :return:  the prediction at a specific index
+        """
         if (len(self.last_word_model_preds) <= pred_index):
             return
         # handle extreme points
         if '?' not in self.last_word_model_preds[pred_index].text:
             return self.last_word_model_preds[pred_index]
 
-        before, after = self._get_before_after(pred_index)
```
Comment (harelm4) [inline src/model/ensemble.py:50]: wasn't used

File: src/model/ensemble.py [hunk 2]
```diff
@@ -100,6 +107,12 @@ def _get_this_word_length(self, index, preds):
         return count
 
     def _get_pred_sequentially_at_index(self, index: int, word_len: int) -> List[Prediction]:
```
Comment (Eldar-Ka) [inline src/model/ensemble.py:109]: looks good

File: src/model/ensemble.py [hunk 3]
```diff
@@ -114,6 +127,11 @@ def _get_pred_sequentially_at_index(self, index: int, word_len: int) -> List[Pre
         return [Prediction(res_txt, round(avg_score, 3))]
 
     def _is_index_starts_a_word(self, pred_index: int) -> bool:
+        """
+        returns true if the prediction at index is a word
```
Comment (Eldar-Ka) [inline src/model/ensemble.py:131]: i.e starts with a space?

**Review by Eldar-Ka — APPROVED** [review]

File: src/model/ensemble.py (hunk 3 above)
Comment (harelm4) [reply src/model/ensemble.py:131]: starts and ends with a space

**Pull request merged**